ansible/templates/whisk.properties.j2

```diff
@@ -53,3 +53,5 @@ use.cli.download=false
 nginx.conf.dir={{ nginx_conf_dir }}
 testing.auth={{ openwhisk_home }}/config/keys/auth.guest
 vcap.services.file=
+limit.entity.actioncode=48m
+limit.entity.parameters=1m
```

Comment:
in #853 the limit is in code right in type. i know this was my suggestion earlier but i think now it was misguided. what do you think since you implemented it and might have a better perspective?